src/Namotion.Reflection/StringBuilderExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Text;
+
+namespace Namotion.Reflection
+{
+    /// <summary>
+    /// Contains extension for <see cref="StringBuilder"/>.
+    /// </summary>
+    public static class StringBuilderExtensions
+    {
+        /// <summary>
+        /// Allows to append multiple strings to the <see cref="StringBuilder"/> at once.
+        /// </summary>
+        /// <remarks>
+        /// Only strings that are neither <c>null</c> nor <c>string.Empty</c> will be added.
+        /// </remarks>
+        /// <param name="stringBuilder">The instance of <see cref="StringBuilder"/>.</param>
+        /// <param name="values">The values to appends.</param>
+        /// <returns>The value of <paramref name="stringBuilder"/>.</returns>
+        public static StringBuilder Append(this StringBuilder stringBuilder, params string[] values)
```

Comment:
this will allocate new array every time when called, might be beneficial to add overloads for common cases used like 1, 2, 3, 4 items which are taken as separate parameters.

Reply:
Added another `Append` method, that takes up to six strings as separate parameters.
Values 3 to 6 are optional but 1 and 2 are mandatory. This is because `StringBuilder` already has an `Append` method that takes a string, so at least two strings must be specified for this method to be called.